### PR TITLE
Host Grotesk: Version 1.003 added

### PR DESCRIPTION
**Pull request by vv-monsalve**
Taken from the upstream repo https://github.com/Element-Type/HostGrotesk at commit https://github.com/Element-Type/HostGrotesk/commit/3879b328f9f7c5dbff4436865271a254e3c06221.
## PR Checklist:

- [x] `minisite_url` definition in the METADATA.pb file for commissioned projects
- [x] `primary_script` definition in the METADATA.pb file for all projects that have a primary non-Latin based language support target
- [x] `subsets` definitions in the METADATA.pb reflect the actual subsets and languages present in the font files (in alphabetic order). For **CJK fonts**, only include one of the following subsets `chinese-hongkong`, `chinese-simplified`, `chinese-traditional`, `korean`, `japanese`.
- [ ] Fontbakery checks are reviewed and failing checks are resolved in collaboration with the upstream font development team
- [ ] Diffenator2 regression checks for revisions on all projects that are currently in production
- [x] Designers bio info have to be present in the designer catalog (at least an issue should be opened for tracking this, if they are not)
- [x] Check designers order in metadata.pb, since the first one of the list appears as “principal designer”
- [ ] Social media formatted visual assets for all new commissioned projects in the Drive directory, communicate with the repository Maintainer so that they can push this content to the Social Media tracker spreadsheet
- [ ] Social media content draft for all new commissioned projects in the Drive directory and Social Media tracker spreadsheet, communicate with the repository Maintainer so that they can push this content to the Social Media tracker spreadsheet

### CODE REVIEW

**Comment by vv-monsalve** [comment]
@emmamarichal 

- The designers for this family are tracked here #8236 
- And the tags here #8257

**Comment by emmamarichal** [comment]
Hi @vv-monsalve! Here is my review:

**Article & Metadatas** (see my comments above)
- 2 articles?
- Designers order
- 'A' instead of 'an'
- Licence URL link

**Fontbakery question:**
- `⚠️ WARN Does the font contain a soft hyphen? `
Only the italic has it, making the glyphset different. If the designers have some changes to do, they can take the opportunity to remove it in italics.

**An weird ogonek issue:**
I already had this issue last week with another font. The 'ogonek' (legacy), appears only in the light master. I investigated but I couldn't find why, since the component looked okay in the sources. The solution I found to avoid that problem was to decompose it in sources.
It works on Indesign however 🤔

- on -Fontgauntlet:

https://github.com/user-attachments/assets/43d2960d-2008-4e90-b988-cee71b117917


- on TextEdit:
Screenshot 2024-10-04 at 11 05 57
Screenshot 2024-10-04 at 11 06 07


**Accents alignements**
- Not a big blocker for me, but the accent positions could be improved here:

Screenshot 2024-10-04 at 10 40 22
Screenshot 2024-10-04 at 11 30 53

**Licence URL**
I've updated it in the OFL.txt, but we still have the old link in the sources:

Screenshot 2024-10-04 at 10 33 47

**No top anchors on combaccents**
- I report it since I don't know if it's intentional:

Screenshot 2024-10-04 at 10 47 05

**Comment by vv-monsalve** [comment]
I'll inspect the essential issues (license + legacy ogonek issue)  and create an upstream PR. Vietnamese (top anchors) is not among the base requirements for submitted fonts. We can go ahead with it as it's now.

**Comment by emmamarichal** [comment]
@vv-monsalve alright, ping me once it's ready :)

File: ofl/hostgrotesk/METADATA.pb [hunk 1]
```diff
@@ -0,0 +1,56 @@
+name: "Host Grotesk"
+designer: "Doğukan Karapınar, Element Type, İbrahim Kaçtıoğlu"
```
[the remark this answers is not in the document]
Comment (vv-monsalve) [reply ofl/hostgrotesk/METADATA.pb:2]: Not really, it is more related to how they appear in the source file. But I'm confirming about this with the designers.

File: ofl/hostgrotesk/article/ARTICLE.en_us.html [hunk 2]
```diff
@@ -0,0 +1,21 @@
+<p>
+    Host Grotesk, an uniwidth sans-serif typeface based on Indian Type Foundry's Poppins (2020). Host Grotesk is a variable font tailored by Elemnt Type for modern user interfaces, ensures uniform letter width and spacing across its weights and corresponding italics, aiming for seamless adaptability without compromising layout consistency.
+</p>
+<p>
+    To contribute, see <a href="//github.com/Element-Type/HostGrotesk">github.com/Element-Type/HostGrotesk</a>
+</p>
+<hr>
+<h2>An uniwidth typeface</h2>
```
[the remark this answers is not in the document]
Comment (vv-monsalve) [reply ofl/hostgrotesk/article/ARTICLE.en_us.html:8]: You already changed it. Tks.

**Comment by vv-monsalve** [comment]
Updated

**Comment by vv-monsalve** [comment]
> Designer Element Type still does not have an avatar image. Please provide one.

Done in #8301 

> It seems that Doğukan Karapınar is still not listed on the designers catalog.

The other designers were added by #8258 and #8234

File: ofl/hostgrotesk/METADATA.pb (hunk 1 above)
[the remark this answers is not in the document]
Comment (vv-monsalve) [reply ofl/hostgrotesk/METADATA.pb]: Solved now.

**Review by emmamarichal — APPROVED** [review]

**Pull request merged**